code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -383,8 +383,8 @@ Command action procs
 		to_chat(user, "The emergency shuttle may not be sent at this time. Please try again later.")
 		return 0
 
-	if(world.time < 6000) // Ten minute grace period to let the game get going without lolmetagaming. -- TLE
-		to_chat(user, "The emergency shuttle is refueling. Please wait another [round((6000-world.time)/600)] minute\s before trying again.")
+	if(world.time < 36000) //Shuttle can now only be recalled after one hour into the round.
```

Comment:
Would be better to make that a config var, so it can easily be decreased for local testing

Reply:
Agreed

Reply:
agreed

Reply:
Done.